themes/bootstrap3/templates/RecordTab/similaritemscarousel.phtml

```diff
@@ -17,7 +17,7 @@
         <?php foreach ($similarRecords as $index => $data): ?>
           <div class="carousel-item">
             <a class="hover-overlay record-link" href="<?=$this->recordLink()->getUrl($data)?>">
-              <?=$this->record($data)->getCoverDetails('core', 'large')['html']?>
+              <?=$this->record($data)->getCoverDetails('similar-items', 'large')['html']?>
```

Comment:
Thanks for catching this; not sure why this was using "core." I might split this out as a separate commit before merging the rest of the PR.

Reply:
Okay, I have done this (commit 3b36397).